src/tests/lib/qunit_config.js

```diff
@@ -1 +1,41 @@
-QUnit.config.testTimeout = 6400;
+/* globals badger:false */
+
+(function () {
+
+  let BACKUP = {};
+
+  QUnit.config.testTimeout = 6400;
+
+  // runs before any tests begin
+  QUnit.begin(() => {
+    // disable storage persistence
+    chrome.storage.local.set = () => {};
+
+    // set defaults
+    var settings = badger.storage.getBadgerStorageObject("settings_map");
+    for (let key in badger.defaultSettings) {
+      settings.setItem(key, badger.defaultSettings[key]);
+    }
+  });
+
+  // runs after all tests finished
+  QUnit.done(() => {
+  });
+
+  QUnit.testStart(() => {
```

Comment:
No, we need to reset state between tests. See how several tests fail if you move this to `QUnit.begin()`.